JASP-Desktop/po/jasp_nl.po

```diff
@@ -20001,239 +20007,224 @@ msgctxt "Description|"
 msgid "Visual Modeling"
 msgstr "Visueel modelleren"
 
-#, fuzzy
 msgctxt "LSbinomialestimation|"
 msgid "Color palette"
 msgstr "Kleurenpalet"
 
-#, fuzzy
 msgctxt "LSbinomialestimation|"
 msgid "Colorblind"
 msgstr "Kleurenblind"
 
-#, fuzzy
 msgctxt "LSbinomialestimation|"
 msgid "Colorblind Alt."
-msgstr "Kleurenblind alt."
+msgstr "Kleurenblind Alt."
 
-#, fuzzy
 msgctxt "LSbinomialestimation|"
 msgid "Viridis"
 msgstr "Viridis"
 
-#, fuzzy
 msgctxt "LSbinomialestimation|"
 msgid "ggplot2"
 msgstr "ggplot2"
 
-#, fuzzy
 msgctxt "LSbinomialestimation|"
 msgid "Gray"
 msgstr "Grijs"
 
 msgctxt "LSbinomialestimation|"
 msgid "Model "
-msgstr ""
+msgstr "Model "
 
-#, fuzzy
 msgctxt "LSbinomialtesting|"
 msgid "Color palette"
 msgstr "Kleurenpalet"
 
-#, fuzzy
 msgctxt "LSbinomialtesting|"
 msgid "Colorblind"
 msgstr "Kleurenblind"
 
-#, fuzzy
 msgctxt "LSbinomialtesting|"
 msgid "Colorblind Alt."
-msgstr "Kleurenblind alt."
+msgstr "Kleurenblind Alt."
 
-#, fuzzy
 msgctxt "LSbinomialtesting|"
 msgid "Viridis"
 msgstr "Viridis"
 
-#, fuzzy
 msgctxt "LSbinomialtesting|"
 msgid "ggplot2"
 msgstr "ggplot2"
 
-#, fuzzy
 msgctxt "LSbinomialtesting|"
 msgid "Gray"
 msgstr "Grijs"
 
 msgctxt "LSbinomialtesting|"
 msgid "Hypothesis "
-msgstr ""
+msgstr "Hypothese "
 
 msgctxt "LSbinomialdatainput|"
 msgid "Input Type"
-msgstr ""
+msgstr "Invoer Type"
 
 msgctxt "LSbinomialdatainput|"
 msgid "Count Data"
-msgstr ""
+msgstr "Tellingen"
 
 msgctxt "LSbinomialdatainput|"
 msgid "Comma-separated Sequence of Observations"
-msgstr ""
+msgstr "Door komma's gescheiden reeks waarnemingen"
 
-#, fuzzy
 msgctxt "LSestimationinference|"
 msgid "Mass"
 msgstr "Massa"
 
 msgctxt "LSestimationinference|"
 msgid "Lower"
-msgstr ""
+msgstr "Onder"
 
 msgctxt "LSestimationinference|"
 msgid "Upper"
-msgstr ""
+msgstr "Boven"
 
 msgctxt "LSestimationpredictions|"
 msgid "Posterior Prediction"
-msgstr ""
+msgstr "Posterior Voorspelling"
 
 msgctxt "LSestimationpredictions|"
 msgid "Number of future trials"
-msgstr ""
+msgstr "Aantal toekomstige proeven"
```

Comment:
```suggestion
msgstr "Aantal toekomstige steekproeven"
```
of vertalen we dit in andere analyses ook zo?

Reply:
Wordt door elkaar gebruikt. In desktop wordt meer 'proeven' gebruikt.
Consequent 'steekproef' lijkt mij dan beter.
@AlexanderLyNL wat vind jij?

Reply:
Toekomstige steekproefgrootte